index.html

```diff
@@ -39,6 +46,7 @@ <h2>LET’S CONNECT</h2>
     <section class="allbocks">
```

Comment:
- [ ] you did good job to implement this section but still there is something you missed out kindly consider checking on the below screenshots 
- your design 👇 
![nn](https://user-images.githubusercontent.com/60146030/220759248-eb6639a5-0632-4ec7-b2f4-9f64bc922fd4.PNG)
- Figma design 👇
![ff](https://user-images.githubusercontent.com/60146030/220759589-311fe398-891a-4a62-ae3a-99ff12917dd3.PNG)

kindly consider making the left space equal to the right space to make a good copy of the Figma Design